roles/contiv_network/tasks/ovs_cleanup.yml

```diff
@@ -28,6 +28,7 @@
 - name: deny openvswitch_t type in selinux
   shell: >
       semanage permissive -d openvswitch_t
+  when: (ansible_os_family == "RedHat")
```

Comment:
> why ?

this is to be consistent with rest of the cleanup steps. We don't need conditionals unless we absolutely need them.